smartcontractkit/external-adapters-js
# Feature/df 18971 gsr lwba endpoint

## Closes DF-18971

## Description
Added GSR LWBA endpoint

## Changes
- Added `crypto-lwba` endpoint as alias to default `price` endpoint with additional aliases `'cryptolwba', and 'crypto_lwba'`
- Added expected LWBA fields to the DP and EA endpoint response bodies in respective `endpoint/` and `transport/` dirs
- Refactored GSR auth code into separate `authutils.ts` file with minor changes to accept multiple sets of configs
- Updated integration tests



## Steps to Test

1. GSR price endpoint regression (related to refactored authutils)
2. New LWBA endpoint returns expected fields corresponding to DP fields
3. **Make sure to use the Mercury Prod credentials as the bid/ask fields are only included for specific credentials**

## Quality Assurance
